aspnetcore/migration/inc/overview.md

```diff
@@ -0,0 +1,70 @@
+uid: migration/inc/overview
+---
+
+# Incremental ASP.NET to ASP.NET Core migration
+
+Migrating an app from ASP.NET Framework to ASP.NET Core is non-trivial for the majority of production apps. These apps often incorporate new technologies as they become available and are often composed of many legacy decisions. This article provide guidance and links to tools for migrating ASP.NET Framework apps to ASP.NET Core with as little change as possible.
+
+One of the larger challenges is the pervasive use of <xref:System.Web.HttpContext> throughout a code base. Without the incremental approach and tools, a large scale rewrite is required to remove the `System.Web.HttpContext` dependency. The adapters in [dotnet/systemweb-adapters](https://github.com/dotnet/systemweb-adapters) provide a set of runtime helpers to access the types the ASP.NET Framework app used but in a way that works using ASP.NET Core with minimal changes.
+
+A complete migration may take considerable effort depending on the size of the app, dependencies, and non-portable APIs used. In order to keep deploying an app to production while working on migrating, the best pattern is to follow is the [Strangler Fig pattern](/azure/architecture/patterns/strangler-fig). The *Strangler Fig pattern* allows for continual development on the old system with an incremental approach to replacing specific pieces of functionality with new services. This document describes how to apply the Strangler Fig pattern to an ASP.NET app migrating towards ASP.NET Core.
+
+If you'd like to skip this over and get started, see [Get started](xref:migration/inc/start).
+
+## App migration to ASP.NET Core
+
+Before starting the migration, the app targets ASP.NET Framework and runs on Windows with its supporting libraries:
+
+![Before starting the migration](~/migration/inc/overview/static/1.png)
+
+Migration starts by introducing a new app based on ASP.NET Core that becomes the entry point. Incoming requests go to the ASP.NET Core app, which either handles the request or proxies the request to the .NET Framework app via [YARP](https://microsoft.github.io/reverse-proxy/). At first, the majority of code providing responses is the .NET Framework app, but the ASP.NET Core app is now set up to start migrating routes:
+
+![start migrating routes](~/migration/inc/overview/static/nop.png)
+
+To migrate business logic that relies on `HttpContext`, the libraries need to be built with `Microsoft.AspNetCore.SystemWebAdapters`. Building the libraries with `Microsoft.AspNetCore.SystemWebAdapters` allows:
+
+* The libraries to be built against .NET Framework, .NET Core, or .NET Standard 2.0.
+* Ensures that the libraries are using APIs that are available on both ASP.NET and ASP.NET Core.
+
+![Microsoft.AspNetCore.SystemWebAdapters](~/migration/inc/overview/static/sys_adapt.png)
+
+<!-- Review: Why does this need to be serialized? Can't one team migrate WebAPI, another specific controllers, another, ASPX pages, etc -->
```

Comment:
Review

Reply:
Once the set up is done, there is no need to serialize the migration of any endpoint - they can be done in parallel by different teams if needed